addons/website/static/src/xml/website.editor.xml

```diff
@@ -37,11 +37,11 @@
         <div class="form-row justify-content-between o_options_container"/>
     </div>
     <t t-name="website.theme_customize_modal_option">
-        <a t-if="font or label" href="#" t-attf-class="dropdown-item p-0 o_theme_customize_option #{font ? 'o_theme_customize_option_font_' + font : ''}">
-           <label class="justify-content-start" t-att-data-font-id="font" t-att-data-label="label">
```

Comment:
@kma-odoo Better but still no need of this. This creates duplicated code (for the input, the label, etc). Simply wrap the renderered template in a dropdown-item on the JS side if needed. So, in JS, do:
```javascript
qweb.render('website.theme_customize_modal_option', ...);
if (selection) {
    // wrap in a dropdown-item
}
```
This way, it is just adding a few JS line instead of duplicating most of the qweb rendering :)